src/org/thoughtcrime/securesms/RegistrationActivity.java

```diff
@@ -175,6 +176,17 @@ public void onTextChanged(CharSequence s, int start, int before, int count) { }
 
             int certCheckFlags = DcHelper.getInt(this, "imap_certificate_checks");
             certCheck.setSelection(certCheckFlags);
+        } else if (getIntent() != null && getIntent().getBundleExtra(ACCOUNT_DATA) != null) {
+          // Companion app might have sent account data
+          Bundle b = getIntent().getBundleExtra(ACCOUNT_DATA);
+          String emailAddress = b.getString(CONFIG_ADDRESS);
+          String password = b.getString(CONFIG_MAIL_PASSWORD);
+          if (!TextUtils.isEmpty(emailAddress) && !TextUtils.isEmpty(password)) {
+            emailInput.setText(emailAddress);
+            passwordInput.setText(password);
+            updateProviderInfo();
```

Comment:
It seems to work also without that line and that would reduce the visual noise a little bit. I'm not sure of all consequences though. @r10s maybe you can have a closer look here?

Reply:
you are taking about the single statement `updateProviderInfo()`?

i had a look, it should be okay to not call it, it only updates the ui and should not affect login.

Reply:
yes that line.
I would leave that out then. the yellow info box disappears way too fast to read it's content when yggmail is configured automatically.

Reply:
yip, better leave out the call to `updateProviderInfo()` then.